peekingduck/pipeline/nodes/model/posenetv1/posenet_files/detector.py

```diff
@@ -1,5 +1,5 @@
 # Copyright 2018 Ross Wightman
-# Modifications copyright 2021 AI Singapore
+# Modifications copyright 2022 AI Singapore
 # Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Add a vertical whitespace between these lines, follow the format of peekingduck/pipeline/nodes/model/__init__.py